src/Phinx/Db/Adapter/SqlServerAdapter.php

```diff
@@ -723,11 +723,11 @@ protected function getDefaultConstraint(string $tableName, string $columnName)
     }
 
     /**
-     * @param int $tableId Table ID
-     * @param int $indexId Index ID
+     * @param string $tableId Table ID
+     * @param string $indexId Index ID
      * @return array
      */
-    protected function getIndexColums(int $tableId, int $indexId): array
+    protected function getIndexColums(string $tableId, string $indexId): array
```

Comment:
This is passed strings not ints and is internal to this adapter so changed the types.

Reply:
An unfortunate "quirk" of PDO in that it returns numerical columns as strings.